### PR TITLE
Answer 'How can I start using conda?' for FAQ doc

### LINKED ISSUE #47: FAQ - Add How can I start using conda? question
## Description

Add the question "How can I start using conda?" to the conda.org FAQ (docs/faq.md) and answer it. This answer could start with "There are several ways of getting conda in your computer... (link to Installation page)"

### PR DESCRIPTION
**Pull request by deepeshaburse**
Fixes #47 

PR to add the answer to the [FAQ "How can I start using conda?"](https://github.com/conda-incubator/conda-dot-org/issues/47)

### CODE REVIEW

File: learn/faq.md [hunk 1]
```diff
@@ -78,4 +78,6 @@ If you need help with any aspect of the conda ecosystem, feel free to reach out
 
 For more information, read [How to reach us](/community/#how-to-reach-us) on the [Community](/community) page.
 
- 
+## How can I start using conda?
+
+There are several ways to get conda on your computer. You can refer to the [installation guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) for a step by step procedure.
```
Comment (kathatherine) [inline learn/faq.md:83]: This is a good start. As a user of conda, do you think you would want more out of an answer like this? Maybe point them to channel or package resources? Actually, a lot of the other FAQ answers cover things like this, so maybe that's unnecessary. We could point to those questions, though. And to help with that, let's put this question right after "What is a feedstock?"

Comment (kathatherine) [inline learn/faq.md:83]:
```suggestion
There are several ways to get started with conda. See the [installation guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) for step by step instructions. 
```

**Review by kathatherine — CHANGES_REQUESTED** [review]
Thanks for the commit! I have a few suggestions.

File: learn/faq.md (hunk 1 above)
Comment (deepeshaburse) [reply learn/faq.md:83]: Sure, yes. I noticed that we have many questions covering important things I would need as a new user hence didn't add much detail. I'll put this question under "What is feedstock?", see if I can add anything that won't overlap much with another answer and link answers wherever relevant. Thanks!

Comment (deepeshaburse) [reply learn/faq.md]: Hi @kathatherine, please take a look at the changes I have made and let me know if this is what you were expecting. Thank you!

File: learn/faq.md [hunk 2]
```diff
@@ -26,6 +26,10 @@ For more information, please check out the [conda documentation page on channels
 
 A feedstock is a conda package repository.
 
+## How can I start using conda?
+
+There are several ways to get started with [Conda](./faq.md/#what-is-conda). See the [installation guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) for step by step instructions. You can [search for the packages](./faq.md/#how-can-i-search-for-packages) you require and also check out [conda channels](./faq.md/#what-is-a-conda-channel). In case you face any issues, feel free to [contact us](./faq.md/#where-can-i-find-help).
```
[the remark this answers is not in the document]
Comment (kathatherine) [reply learn/faq.md:31]: Conda is lowercase unless starting a sentence. I also tweaked a few sentence-structure things.

**Review by kathatherine — CHANGES_REQUESTED** [review]
This is great. Thank you for this!

File: learn/faq.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (deepeshaburse) [reply learn/faq.md:31]: Noted, thank you for the review!

**Review by kathatherine — APPROVED** [review]
Thank you, looks good to me

**Review by travishathaway — CHANGES_REQUESTED** [review]
I think this should also include a link to install mamba, which is a conda compatible tool. Here is the installation link for that: https://mamba.readthedocs.io/en/latest/installation.html

We have to remember that this is not a website for the conda tool itself but the entire conda packaging ecosystem.

**Comment by deepeshaburse** [comment]
@travishathaway Definitely, I'll add that too. Thank you for the review!

**Pull request closed without merging**